src/main/kotlin/racingcar/domain/car/action/RandomMovingStrategy.kt

```diff
@@ -0,0 +1,9 @@
+package racingcar.domain.car.action
+
+private const val MOVING_MINIMUM_CONDITION_NUMBER = 4
+private const val RANDOM_NUMBER_START_INDEX = 0
+private const val RANDOM_NUMBER_END_INDEX = 9
+
+class RandomMovingStrategy {
```

Comment:
```suggestion
class RandomMovingStrategy:  MovingStrategy {
```
움직임 전략을 상속받는 구조로 구현하는 경험을 해보세요 😄

Reply:
이미해 주신 조언이신데요 😢 제가 기존에 `MovingStrategy` 전략을 상속받는 `MovingStrategyImpl` 클래스명을 `RandomMovingStrategy`으로 변경하면서, 움직임 전략을 상속 받는 부분을 누락한것 같네요 😭
아래와 같이 수정하였습니다!

- https://github.com/next-step/kotlin-racingcar/commit/0bc11de2efe61e0f39030329c153c79841a06c50